### PR TITLE
Removing EKS as open contractor position

### PR DESCRIPTION
**Pull request by gitsstewart**
We hired someone so no longer need to leave the EKS SME role open for applications

### CODE REVIEW

File: pages/careers/_open-positions.html [hunk 1]
```diff
@@ -329,7 +329,7 @@ <h4>What You'll Work On</h4>
     <h4>Your Ideal Background</h4>
     <ul>
       <li>You have a strong background in software engineering, with 5+ years of experience.</li>
-      <li>You have significant production experience with either VPC, EKS, or ECS.</li>
+      <li>You have significant production experience with either VPC, or ECS.</li>
```
Comment (eak12913) [inline pages/careers/_open-positions.html:332]:
```suggestion
      <li>You have significant production experience with either VPC or ECS.</li>
```

File: pages/careers/_open-positions.html [hunk 2]
```diff
@@ -280,7 +280,7 @@ <h4 style="margin-top: 14px;" id="we-are-not-actively-hiring-for-this-role"><a c
       in specific areas of the Gruntwork Library, and help customers succeed with them.
     </p>
     <p>
-      This role will focus specifically on one of three main areas of expertise in our Gruntwork Library: EKS, VPC, or ECS. 
+      This role will focus specifically on one of three main areas of expertise in our Gruntwork Library: VPC, or ECS.
```
Comment (eak12913) [inline pages/careers/_open-positions.html:283]:
```suggestion
      This role will focus specifically on one of three main areas of expertise in our Gruntwork Library: VPC or ECS. 
```

**Review by eak12913 — COMMENTED** [review]
I added a nit suggestion to remove commas when there are just two options. If it was intentional - just lemme know and I'll approve. Otherwise you can just commit those suggestions and I'll approve.

**Review by eak12913 — APPROVED** [review]
👍

**Review by arsci — APPROVED** [review]

**Pull request merged**